src/Donations/resources/components/DonationsListTable.tsx

```diff
@@ -143,6 +144,14 @@ export default function DonationsListTable() {
             filterSettings={filters}
             paymentMode={!!window.GiveDonations.paymentMode}
         >
+            {!!window.GiveDonations.manualDonations && (
```

Comment:
Do we have to convert the boolean value to a boolean again?

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6638/commits/28b8861792a1561b52a06062689a9f0d82a64ff5